index.html

```diff
@@ -8,7 +8,7 @@
     <link
       href="https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300|Sonsie+One"
       rel="stylesheet" />
```

Comment:
[Optional] This font is not being used. It is good practice to only leave resources in your files which are actually serving a purpose, anything else in your code that is not being used should be removed to avoid bugs or confusion